client/src/components/Calendar.jsx

```diff
@@ -176,20 +189,27 @@ class Calendar extends React.Component {
                 <th>Sa</th>
               </tr>
               {this.state.weeks.length
-                ? this.state.weeks.map((week, ind) => <Week key={ind} week={week}/>)
+                ? this.state.weeks.map((week, ind) => 
+                  <Week 
+                    key={ind} 
```

Comment:
What about putting all these props into one object such as reservationInfo and only pass that around to Week and Day components. It'd be less code and easier to read